src/components/label/label.wp.scss

```diff
@@ -61,8 +61,8 @@ html[dir="rtl"] .label-wp[floating] {
   transform: translate3d(0, 0, 0) scale(.8);
 }
 
-html[dir="rtl"] .input-has-focus .label-wp[floating],
-html[dir="rtl"] .input-has-value .label-wp[floating] {
+[dir="rtl"] .input-has-focus .label-wp[floating],
```

Comment:
Hmm.. ok.
I might nor know enough about CSS priorities, but from a small test, it seems like it goes like I said it would: https://codepen.io/anon/pen/BRzxza

As far as I understand, that is because of the order of selectors:
`.input-has-focus .label-wp[floating]` is after `[dir="rtl"] .label-wp[floating]` and thus takes president.

Let me know if you still think otherwise (as again, I am not completely sure of it)